src/clib/pioc_sc.c

```diff
@@ -213,6 +222,13 @@ PIO_Offset GCDblocksize(int arrlen, const PIO_Offset *arr_in)
             numgaps = ii;
         }
 
+        PIO_Offset *loc_arr = NULL;
+        if (arrlen > 1)
```

Comment:
size of loc_arr depends on "n" not "arrlen"

Reply:
Note that the original code is "PIO_Offset loc_arr[arrlen - 1];"
Also, n is less than or equal to arrlen - 1, and the loop below indicates that size of loc_arr should be "arrlen - 1"
for (int i = 0; i < arrlen - 1; i++)
    if(del_arr[i] != 1)
        loc_arr[j++] = i;

Reply:
ok, thanks (since we are just replacing varlen array code with mallocs, I think this code should be fine)

Reply:
We should also use calloc to initialize loc_arr. Some unit tests show that not all entries are initialized after malloc.